migrations/versions/1020_add_buyer_email_domain_table.py

```diff
@@ -0,0 +1,27 @@
+"""Add buyer email domain table
+
+Revision ID: 1020
+Revises: 1010
+Create Date: 2017-10-10 15:18:22.683693
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1020'
+down_revision = '1010'
+
+
+def upgrade():
+    op.create_table(
+        'buyer_email_domains',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('domain_name', sa.String(length=255), nullable=False),
```

Comment:
Should we add a uniqueness constraint on the domain?

Reply:
We could do - the validation around what counts as an 'existing' domain is a bit more complicated so I thought it might be easier to keep it in the code, but there'd be no harm having `unique=True` as well I suppose.

Reply:
I don't think we ever set a length for our String fields anywhere else - we just have `sa.String()` in our migrations.  I don't see any reason to be explicit here?  Varchar FTW!

Reply:
That was auto-generated by Alembic because it was on my model, which I copied the structure of from the most recent model i.e. `DirectAwardProject` 😸 We seem to specify `String(255)` for a lot of our `name` columns but I agree that it doesn't seem necessary here. Happy to remove!